lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -119,31 +124,37 @@ class FeatureCard extends React.Component {
         }, 1000);
 
         this.setState({ //TODO: Fancy animation
-          signup_animation: 'tada'
+          signupAnimation: 'tada'
         });
       }
     }));
   }
 
   viewAnother() {
-    let newIndex = this.state.current_campaign_index + 1;
-    if (newIndex >= this.state.campaigns.length) {
-      newIndex = 0;
-    }
+    // Remove current campaign
+    this.state.campaigns.splice(0, 1);
```

Comment:
The problem I was running into was we need to remove campaigns from the campaign array when you sign up. If you do that the state changes & things get out of sync

I figured I could either remove the index or store campaigns off the state, wasn't sure which was better.